### DISCUSSION TITLE
Runner and DinD utilize very little CPU

### DISCUSSION THREAD

**Discussion by rarecrumb** [actions/actions-runner-controller#3599] [Questions]
Using self-hosted runners on EKS in `dind` mode, I'm building docker images with nodejs / nextjs, using npm and yarn, etc.

What I see is that even though I request 1 or 2 cores of CPU in the containers, I see my pods regularly only using 6m cpu and 300Mi of memory.

Is there anything I can do to make my build use more resources?

Just using the standard `docker/build-push-action@v5` action...

**Comment by erichorwath** [comment]
maybe it's a bug on how the metrics are shown: https://github.com/actions/actions-runner-controller/issues/2284
